Automattic/studio
# Skip plugins and themes in internal Studio wp-cli commands like import export

## Related issues

- https://github.com/Automattic/dotcom-forge/issues/8961
- https://github.com/Automattic/wp-cli-sqlite-command/pull/7#issuecomment-2548389064
- https://github.com/Automattic/dotcom-forge/issues/10059#issuecomment-2546362614

## Proposed Changes

- Let's skip plugins and themes executing wp-cli for Studio internal commands.
- Keep the regular wp-cli behaviour for commands executed by the assistant 

## Testing Instructions


- Apply this diff to log the arguments being executed:
```
diff --git a/src/site-server.ts b/src/site-server.ts
index 102b650..4371b19 100644
--- a/src/site-server.ts
+++ b/src/site-server.ts
@@ -191,6 +191,8 @@ export class SiteServer {
                        wpCliArgs.push( '--skip-themes' );
                }
 
+               console.log( '\n\n\n\n wpCliArgs', wpCliArgs );
+
                // The parsing of arguments can include shell operators like `>` or `||` that the app don't support.
                const isValidCommand = wpCliArgs.every(
                        ( arg: unknown ) => typeof arg === 'string' || arg instanceof String
```

- Run `npm start`
- Create a Studio site
- Open the terminal
- Run: `wp plugin install slider-revolution-search-replace --activate`
- Go to the assistant tab
- Ask "what plugins do I have installed?"
- Observe the answer is correct
- Observe in the node console the `--skip-plugins` and `--skip-themes` are not present
- Go to the sync tab
- Connect a site
- Push the site
- Observe the push successfully works
- Observe in the node console the `--skip-plugins` and `--skip-themes` are added to each wp-cli command to export, get site url, etc.

## Pre-merge Checklist
